### PR TITLE
Update kotlin-osgi.md

### PR DESCRIPTION
**Pull request by DRSchlaubi**
- Add Kotlin DSL examples
- Use current Kotlin Version in Groovy example
- Do not use `compile` dependency config

### CODE REVIEW

File: docs/topics/kotlin-osgi.md [hunk 1]
```diff
@@ -40,13 +40,42 @@ To exclude the standard library from external libraries (notice that "star exclu
...
 To exclude default Kotlin libraries that comes as transitive dependencies you can use the following approach:
 
+<tabs group="build-script">
+<tab title="Kotlin" group-key="kotlin">
+
+```kotlin
+dependencies {
+ compile("some.group.id", "some.library", "someversion"]) {
+  exclude(group = "org.jetbrains.kotlin")
+}
```
Comment (Tapchicoma) [inline docs/topics/kotlin-osgi.md:73]:
```suggestion
 implementation("some.group.id:some.library:someversion") {
  exclude(group = "org.jetbrains.kotlin")
}
```

File: docs/topics/kotlin-osgi.md [hunk 2]
```diff
@@ -40,13 +40,42 @@ To exclude the standard library from external libraries (notice that "star exclu
...
+```
+
+</tab>
+<tab title="Groovy" group-key="groovy">
+
 ```groovy
 dependencies {
  compile (
```
Comment (Tapchicoma) [inline docs/topics/kotlin-osgi.md:81]: Here code should also be updated - `compile` -> `implementation`, dependency notation as single string

File: docs/topics/kotlin-osgi.md [hunk 3]
```diff
@@ -40,22 +40,52 @@ To exclude the standard library from external libraries (notice that "star exclu
...
+
+</tab>
+<tab title="Groovy" group-key="groovy">
+
 ```groovy
 dependencies {
- compile (
-   [group: 'some.group.id', name: 'some.library', version: 'someversion'],
-   .....) {
+ compile('some.group.id:some.library:someversion') {
```
Comment (Tapchicoma) [inline docs/topics/kotlin-osgi.md:81]: Please change `compile` into `implementation`

Comment (Tapchicoma) [inline docs/topics/kotlin-osgi.md:81]:
```suggestion
 implementation('some.group.id:some.library:someversion') {
```

**Review by Tapchicoma — APPROVED** [review]

File: docs/topics/kotlin-osgi.md [hunk 4]
```diff
@@ -40,22 +40,52 @@ To exclude the standard library from external libraries (notice that "star exclu
 ## Gradle
 
 To include `kotlin-osgi-bundle` to a Gradle project:
+<tabs group="build-script">
+<tab title="Kotlin" group-key="kotlin">
+
+```kotlin
+dependencies {
+  implementation(kotlin("osgi-bundle"))
```
Comment (PetrakovichVictoria) [inline docs/topics/kotlin-osgi.md:48]:
```suggestion
    implementation(kotlin("osgi-bundle"))
```

File: docs/topics/kotlin-osgi.md [hunk 5]
```diff
@@ -40,22 +40,52 @@ To exclude the standard library from external libraries (notice that "star exclu
...
+</tabs>
+
 To exclude default Kotlin libraries that comes as transitive dependencies you can use the following approach:
 
+<tabs group="build-script">
+<tab title="Kotlin" group-key="kotlin">
+
+```kotlin
+dependencies {
+ implementation("some.group.id:some.library:someversion") {
```
Comment (PetrakovichVictoria) [inline docs/topics/kotlin-osgi.md:71]:
```suggestion
    implementation("some.group.id:some.library:someversion") {
```

File: docs/topics/kotlin-osgi.md [hunk 6]
```diff
@@ -40,22 +40,52 @@ To exclude the standard library from external libraries (notice that "star exclu
...
+  exclude(group = "org.jetbrains.kotlin")
```
Comment (PetrakovichVictoria) [inline docs/topics/kotlin-osgi.md:72]:
```suggestion
        exclude(group = "org.jetbrains.kotlin")
```

**Comment by PetrakovichVictoria** [comment]
Hi @DRSchlaubi, thank you for your work!

**Pull request merged**